src/app/Components/header/header.component.ts

```diff
@@ -20,10 +21,12 @@ export class HeaderComponent implements OnInit {
   searchValue = '';
   name$!: Observable<string>;
   token: string | null = null;
+  search: boolean = false;
```

Comment:
tên rõ ràng chức năng của nó nhé
```suggestion
  isDisplaySearch: boolean = false;
```

Reply:
@tritdm121099 dạ em fix rồi ạ